src/common.rs

```diff
@@ -156,6 +195,10 @@ macro_rules! boilerplate_fft_oop {
             fn get_outofplace_scratch_len(&self) -> usize {
                 self.outofplace_scratch_len()
             }
+            #[inline(always)]
+            fn get_immutable_scratch_len(&self) -> usize {
+                self.inplace_scratch_len()
```

Comment:
I don't think this is correct. There are four uses of this macro - Dft, Radix3, Radix4, RadixN

For Dft, it's just 0. For Radix#, it's `inner_fft.len()`. IMO this should call `self.immut_scratch_len()` so we can customize these.

Reply:
Yes, that's perfect